lib/provider.js

```diff
@@ -33,14 +33,18 @@ module.exports = {
 
   getSuggestions({editor, bufferPosition, scopeDescriptor}) {
     const prefix = getPrefix(editor, bufferPosition);
+    const filePath = editor.getPath();
 
     if (shouldPreventAutocomplete(prefix)) {
       return null;
     }
 
+    if (filePath === undefined) {
```

Comment:
Sounds good. Will do.

Though I do like to check for `undefined` explicitly since a lot of things are falsy in JS, e.g. an `''` or `0`.